### ISSUE TITLE
get an error if using the manual configuration

### ISSUE THREAD

**Issue by juggledad** [insectos/node-red-ssh-exec#4]
**Describe the bug**
using the demo flow I set ssh-exec  and use the 'not manual' option with name of one of my pi's and it's password. 
when deploying the ssh-exec node shows a red triangle and 'disconnected' If the flow is run an error occurs:

```
9/6/2023, 6:58:09 AM[node: debug from catch node](http://localhost:1880/#)
msg : Object
object
errMsg: "Error: All configured authentication methods failed"
host: undefined
error: object
message: "Connection error"
source: object
id: "4dcea860ab524b27"
type: "ssh-exec"
name: undefined
count: 1
_msgid: "ce442bc7ce61e351"

```
**To Reproduce**
Do what I stated above

**Expected behavior**
I would expect it to work since there is nothing in the help saying it wouldn't

**Comment by Stwissel** [comment]
could you provide a reproducer (obviously not including your password ;-) )?

**Comment by Stwissel** [comment]
fixed in 0.3.2
